src/Types/ProcessingFigures.tsx

```diff
@@ -41,13 +42,14 @@ export class AnimatedFigure {
       this.velocity = p5.createVector(3 * this.randSign(), 3 * this.randSign());
       this.force = p5.createVector(0, 0); 
       this.mass = Math.random() * 0.03 + 0.003;
-      this.acceleration = p5.createVector(0,0);
+      this.acceleration = p5.createVector(0, 0.5);
       this.angle = 0.0;
       this.timer = 60;
       this.spin = 0.02*this.randSign();
       this.dead = false;
       this.rotAngle = 0.0;
       this.color = c;
+      this.opacity = 200;

```

Comment:
I'm not sure I like a default opacity of <255, but it can easily be changed later so its not really an issue.